addons/portal/controllers/portal_thread.py

```diff
@@ -45,7 +45,7 @@ def portal_chatter_init(self, thread_model, thread_id, **kwargs):
     def _store_portal_thread_fields(cls, res: Store.FieldList, **kwargs):
         portal_partner_by_thread = {
             thread: get_portal_partner(
-                thread, kwargs.get("hash"), kwargs.get("pid"), kwargs.get("token"),
+                thread.sudo(), kwargs.get("hash"), kwargs.get("pid"), kwargs.get("token"),
```

Comment:
We try to always add a small comment above sudo line to explain why they are necessary and why it is acceptable.